backend/src/main/java/gov/cdc/usds/simplereport/service/DeviceTypeService.java

```diff
@@ -182,6 +199,24 @@ public DeviceSpecimenTypeHolder getTypesForFacility(
     return new DeviceSpecimenTypeHolder(defaultType, configuredTypes);
   }
 
+  public DeviceSpecimenTypeHolder getDeviceSpecimenTypesForFacility(
+      String defaultDeviceTypeId, List<String> configuredDeviceSpecimenTypeIds) {
```

Comment:
**suggestion:** This `String` should probably be a UUID – it would avoid the awkward `toString()` on line 210 here

Reply:
See https://github.com/CDCgov/prime-simplereport/pull/2687/files/4faa24fa1b8bc8cbd2e3c78610a3f4b34101f91b#r723618003.